network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -85,7 +85,7 @@ <h1>{{ block.value }}</h1>
           col-12
         {% endif %}"
       >
-        {% include "./fragments/ih_cta.html" %}
+        {% include "./fragments/ih_cta.html" with wrapper_class="my-5" %}
```

Comment:
At first glance this seems outside the scope of this PR- what's going on?

Reply:
Agreed, let's make this a tiny follow-up specific to adding the correct wrapper class for campaign pages.

Reply:
The `ih_cta` section came with default `my-5` margin. I had to make these code changes here to make the `ih_cta` section on blog page style properly (`95px` as top margin and with no bottom margin). Bootstrap classes come with `!important` in the rules so I can't override the margins unless I include `!important` in the SCSS code I'm writing for blog post page's `ih_cta`. I'd like to avoid using `!important` to keep our SCSS code sane.

Reply:
If we are okay with letting `ih_cta` use the default style on blog page for now then I'll revert my changes and fix styling as follow-up.

Reply:
Yeah, I'm with Pomax- I think this needs a little more attention and I'm not convinced this is the right approach for it. Back it out just to reduce the complexity of this PR a bit.

Reply:
follow-up ticket filed: https://github.com/mozilla/foundation.mozilla.org/issues/3111